htdocs/societe/class/api_contacts.class.php

```diff
@@ -78,6 +78,7 @@ public function get($id, $includecount = 0)
 			$result = $this->contact->initAsSpecimen();
 		} else {
 			$result = $this->contact->fetch($id);
+			$this->contact->fetchRoles();
```

Comment:
Because API must be the more atomic and faster than possible, i think it should be better to add a parameter $incuderoles = 0 in parameters of the get. To avoid a subrequest for an unitary API whose goal is to retreive on record into a database.

Reply:
@eldy changes are done :)

Reply:
@ptibogxiv can you keep order of the two original parameters and add includeroles at last position instead in the middle ?

Reply:
@frederic34 done ;)